qa/L0_metrics/general_metrics_test.py

```diff
@@ -62,10 +62,10 @@ def get_model_load_times():
 def load_model_explicit(model_name, server_url="http://localhost:8000"):
     endpoint = f"{server_url}/v2/repository/models/{model_name}/load"
     response = requests.post(endpoint)
-
-    if response.status_code == 200:
+    try:
+        self.assertEqual(response.status_code, 200)
         print(f"Model '{model_name}' loaded successfully.")
-    else:
+    except AssertionError:
```

Comment:
Do we want the test to pass if failed to load the model? If not, you should remove try...except...

Reply:
Yes that's expected behaviour.
Models should load and unload. Else test should fail as subsequent metrics will be incorrect

Reply:
If load or unload failure will result test to fail anyway, why not let it fail at the HTTP response code check instead of metrics check? This way people can easiler identify the root cause of job failure.